app/bundles/StageBundle/Entity/StageRepository.php

```diff
@@ -189,12 +187,15 @@ public function getStageByName($stageName)
         $q->select('partial s.{id, name}')
             ->andWhere($q->expr()->eq('s.isPublished', ':true'))
             ->setParameter('true', true, 'boolean');
-        $q->andWhere(
-            $q->expr()->like('s.name', $stageName)
-        );
+        $q->andWhere("s.name = '".$stageName."'");
```

Comment:
This should probably be set as a parameter since it's coming from user input (CSV) to avoid possible SQL injection.